server/athenian/api/controllers/metrics_controller.py

```diff
@@ -31,16 +31,16 @@
     CalculatedDeploymentMetric, CalculatedDeveloperMetrics, CalculatedDeveloperMetricsItem, \
     CalculatedLinearMetricValues, CalculatedPullRequestMetrics, CalculatedPullRequestMetricsItem, \
     CalculatedReleaseMetric, CodeBypassingPRsMeasurement, CodeCheckMetricsRequest, CodeFilter, \
-    DeploymentMetricsRequest, DeveloperMetricsRequest, ForSet, ForSetCodeChecks, \
-    ForSetDeployments, ForSetDevelopers, PullRequestMetricID, ReleaseMetricsRequest
+    DeploymentMetricsRequest, DeveloperMetricsRequest, ForSetCodeChecks, ForSetDeployments, \
+    ForSetDevelopers, ForSetPullRequests, PullRequestMetricID, ReleaseMetricsRequest
 from athenian.api.models.web.invalid_request_error import InvalidRequestError
 from athenian.api.models.web.pull_request_metrics_request import PullRequestMetricsRequest
 from athenian.api.request import AthenianWebRequest
 from athenian.api.response import model_response, ResponseError
 from athenian.api.tracing import sentry_span
 
 #               service                          developers                                    originals  # noqa
-FilterPRs = Tuple[str, Tuple[List[Set[str]], List[PRParticipants], LabelFilter, JIRAFilter, int, ForSet]]  # noqa
+FilterPRs = Tuple[str, Tuple[List[Set[str]], List[PRParticipants], LabelFilter, JIRAFilter, int, ForSetPullRequests]]  # noqa
```

Comment:
These lines went totally out of control. Let's refactor the types in the future DEV-3968